pkg/naming/env.go

```diff
@@ -1,5 +1,8 @@
 package naming
 
 const (
+	EnvMySQLStateFile    = "MYSQL_STATE_FILE"
+	EnvMySQLNotifySocket = "NOTIFY_SOCKET"
```

Comment:
we should set this like `MYSQL_NOTIFY_SOCKET` do something like https://github.com/percona/percona-xtradb-cluster-operator/blob/main/build/pxc-entrypoint.sh#L302-L303 in entrypoint to make it hard for users to mess up MySQL state manually, otherwise even if you run `mysqld --version` state will change.

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/848/commits/c9337ac39079b20ba6371329951327e7b19aadd2